guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,6 +12,13 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you run {Project} in a disconnected environment, you must obtain and deploy Leapp metadata manually. For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+* You require access to {RHEL} and {Project} packages.
+* Follow the steps for obtaining ISO files for {RHEL} 8 and {Project} in xref:upgrading_a_disconnected_satellite[].
+* Set `module_hotfixes=1` in the repository file when setting up the {Project} repositories.
```

Comment:
I've not tested it in a disconnected setup, but with the most recent `leapp` update we shipped, this one shouldn't be necessary anymore.

Reply:
The comment (even tho GitHub renders is differently) was only meant to the `module_hotfixes` line.